pdns/rec-lua-conf.hh

```diff
@@ -47,6 +47,7 @@ public:
   LuaConfigItems();
   SortList sortlist;
   DNSFilterEngine dfe;
+  std::pair<uint32_t, std::string> trustAnchorFileInfo; // Used to update the Trust Anchors from file periodically
```

Comment:
I would prefer the use of a struct, or perhaps two separate members if that's easier than making a new struct, because otherwise I can't read the code without having to loop up what `first` and `second` are.